cdap-docs/build.sh

```diff
@@ -408,8 +408,16 @@ function build_license_dependency_pdfs() {
 }
 
 function build_standalone() {
```

Comment:
Added code so that it only builds the Hydrator if the directory exists. If not, it prints a message and continues.